app/views/accounts/_form.html.erb

```diff
@@ -155,163 +155,187 @@
     </div>
   <% end %>
 
-  <h2>Rechte im CMS</h2>
-
-  <div class="form-group">
-    <div class="form-check">
-      <%= d.check_box :role_tour, { class: "form-check-input" }, "true", "false" %>
-      <%= d.label :role_tour, "Touren", class: "form-check-label" %>
-    </div>
-  </div>
-
-  <div class="form-group">
-    <div class="form-check">
-      <%= d.check_box :role_point_of_interest, { class: "form-check-input" }, "true", "false" %>
-      <%= d.label :role_point_of_interest, "Orte", class: "form-check-label" %>
-    </div>
-  </div>
+  <h2>Optionen für Resourcen</h2>
 
-  <div class="form-group">
-    <div class="form-check">
-      <%= d.check_box :role_news_item, { class: "form-check-input" }, "true", "false" %>
-      <%= d.label :role_news_item, "Nachrichten", class: "form-check-label" %>
+  <nav>
+    <div class="nav nav-tabs" id="resource-nav-tab" role="tablist">
+      <%= d.fields_for :data_resource_settings do |s| %>
+        <% next unless s.object.data_resource_type %>
+        <a class="nav-item nav-link <%= s.object.data_resource_type == 'Tour' ? 'active' : '' %>" id="nav-contact-tab" data-toggle="tab" href="#nav-<%= s.object.data_resource_type %>" role="tab" aria-controls="nav-contact" aria-selected="false"><%= s.object.data_resource_type.pluralize %></a>
```

Comment:
warum `'Tour'` hier?

Reply:
damit ein Reiter initial aufgeklappt sichtbar ist. Tour ist der erste Reiter in der Liste.
Damit ist der erste Tab initial sichtbar.

Reply:
könnten wir statt hardcoded `Tour` hier programmatisch den titel des ersten resource types entnehmen?

Reply:
Könnten wir die NewsItems als erstes öffnen?